README.md

```diff
@@ -1,9 +1,48 @@
 # CyberSecurity Scoring System (CSSS)
 
+|Branch|Status (Windows Build)|Status (Linux Build)|Last Commit|
+|---|---|---|---|
+|Master|[![AppVeyor Build status](https://img.shields.io/appveyor/ci/stuajnht/CSSS?label=AppVeyor%20Build)](https://ci.appveyor.com/project/stuajnht/csss)|[![Build Status](https://img.shields.io/travis/stuajnht/CSSS.svg?label=Travis%20Build)](https://travis-ci.org/stuajnht/CSSS) |[![Last Master Commit](https://img.shields.io/github/last-commit/stuajnht/CSSS.svg)]()
+|Develop|[![AppVeyor Build status](https://img.shields.io/appveyor/ci/stuajnht/CSSS/develop?label=AppVeyor%20Build)](https://ci.appveyor.com/project/stuajnht/csss/develop)|[![Develop Build Status](https://img.shields.io/travis/stuajnht/CSSS/develop?label=Travis%20Build)]()|[![Last Develop Commit](https://img.shields.io/github/last-commit/stuajnht/CSSS/develop.svg)]()
+
+[![.Net Versions](https://img.shields.io/badge/.Net%20Versions-4.0|4.5|4.6|4.7-blue.svg)]()

```

Comment:
I think .Net 4.8 should also be supported. Please check first before adding!

Reply:
Installed, built & ran tests - can confirm that it appears to work without issue. I'll add it as supported but potentially further work to completely verify testing could be done in a VM.